src/OrchardCore.Modules/OrchardCore.Workflows/Services/WorkflowManager.cs

```diff
@@ -415,6 +417,7 @@ public async Task<IEnumerable<ActivityRecord>> ExecuteWorkflowAsync(WorkflowExec
 
                     // Decrement the workflow scope recursion count.
                     DecrementRecursion(workflowContext.Workflow);
+                    await _workflowFaultHandler.OnWorkflowFaultAsync(this, workflowContext, activityContext, ex);
```

Comment:
Add new line before

Reply:
It's strange that I have submitted the code and it is not updated here??